src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private final double firstBase;
+    private final double secondBase;
+    private final double height;
+
+    public IsoscelesTrapezoid(double firstBase, double secondBase, double height, Color color) {
+
```

Comment:
Delete empty line

Reply:
no ! it will be mistake
'CTOR_DEF' should be separated from previous line. [EmptyLineSeparator]

Reply:
here can't be such mistake 
I check in inteligi maven and he show such type of mistakes

Reply:
delete line No 10